src/utils.ts

```diff
@@ -1981,14 +2066,19 @@ export class newTooltip {
 
   private buildText(context: CanvasRenderingContext2D): [string[], Vertex] {
     context.save();
-    context.font = `${this.fontsize}px sans-serif`
-    let printedRows = ['Information: '];
+    context.font = `${this.fontsize}px sans-serif`;
+    let printedRows = [];
     let textLength = context.measureText(printedRows[0]).width;
```

Comment:
This should break as printedRows is now empty

Reply:
And it does not... i agree with, it is weird.
I put 0 as default value

Reply:
It probably took the length of "undefined"